SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/packages.config

```diff
@@ -1,43 +1,43 @@
+  <package id="SQLitePCLRaw.bundle_green" version="1.1.2" targetFramework="net45" />
+  <package id="SQLitePCLRaw.core" version="1.1.2" targetFramework="net45" />
+  <package id="SQLitePCLRaw.lib.e_sqlite3.linux" version="1.1.2" targetFramework="net45" />
+  <package id="SQLitePCLRaw.lib.e_sqlite3.osx" version="1.1.2" targetFramework="net45" />
+  <package id="SQLitePCLRaw.lib.e_sqlite3.v110_xp" version="1.1.2" targetFramework="net45" />
+  <package id="SQLitePCLRaw.provider.e_sqlite3.net45" version="1.1.2" targetFramework="net45" />
+  <package id="System.Collections" version="4.0.11" targetFramework="net45" />
+  <package id="System.Collections.Concurrent" version="4.0.12" targetFramework="net45" />
+  <package id="System.Diagnostics.Debug" version="4.0.11" targetFramework="net45" />
+  <package id="System.Diagnostics.Tools" version="4.0.1" targetFramework="net45" />
+  <package id="System.Diagnostics.Tracing" version="4.1.0" targetFramework="net45" />
+  <package id="System.Globalization" version="4.3.0" targetFramework="net45" />
+  <package id="System.IO" version="4.1.0" targetFramework="net45" />
+  <package id="System.IO.Compression" version="4.1.0" targetFramework="net45" />
+  <package id="System.Linq" version="4.1.0" targetFramework="net45" />
+  <package id="System.Linq.Expressions" version="4.1.0" targetFramework="net45" />
+  <package id="System.Net.Http" version="4.1.0" targetFramework="net45" />
+  <package id="System.Net.Primitives" version="4.0.11" targetFramework="net45" />
+  <package id="System.ObjectModel" version="4.0.12" targetFramework="net45" />
+  <package id="System.Reflection" version="4.1.0" targetFramework="net45" />
+  <package id="System.Reflection.Extensions" version="4.0.1" targetFramework="net45" />
+  <package id="System.Reflection.Primitives" version="4.0.1" targetFramework="net45" />
+  <package id="System.Resources.ResourceManager" version="4.0.1" targetFramework="net45" />
+  <package id="System.Runtime" version="4.1.0" targetFramework="net45" />
+  <package id="System.Runtime.Extensions" version="4.1.0" targetFramework="net45" />
+  <package id="System.Runtime.InteropServices" version="4.1.0" targetFramework="net45" />
+  <package id="System.Runtime.InteropServices.RuntimeInformation" version="4.3.0" targetFramework="net45" />
+  <package id="System.Runtime.Numerics" version="4.0.1" targetFramework="net45" />
+  <package id="System.Text.Encoding" version="4.0.11" targetFramework="net45" />
+  <package id="System.Text.Encoding.Extensions" version="4.0.11" targetFramework="net45" />
+  <package id="System.Text.RegularExpressions" version="4.1.0" targetFramework="net45" />
+  <package id="System.Threading" version="4.0.11" targetFramework="net45" />
+  <package id="System.Threading.Tasks" version="4.0.11" targetFramework="net45" />
+  <package id="System.Threading.Timer" version="4.0.1" targetFramework="net45" requireReinstallation="true" />
```

Comment:
This is automatically added.
From [nuget doc](https://docs.microsoft.com/en-us/nuget/consume-packages/reinstalling-and-updating-packages#considerations):
> If NuGet detects that packages were affected by retargeting or upgrading a project, it adds a `requireReinstallation="true"` attribute in packages.config to all affected package references. Because of this, each subsequent build in Visual Studio raises build warnings for those packages so you can remember to reinstall them.

Reply:
If it raises build warnings we should do the suggested reinstall?